RELEASE-NOTES.txt

```diff
@@ -1,3 +1,24 @@
+1.3.3
+=====
+* Handle zip imports regression #874
+* Sentry: change release naming #868
+* Sentry: improve breadcrumbs with providing more info #864
+* Sentry: add user identifier #863
+* Show warning when user tries to pull a site that is larger than 2GB #842
+* Remove internal mu-plugins during prepareWordPress #860
+* Include mu-plugins in Studio site import #852
+* Introduced Redux with refactoring ChatProvider as the first store #826
+* Fixed setting environment variables for WP CLI #814
+* Mounted mu-plugin polyfills to the Playground filesystem #845
+* Handled errors of failed opening browser on Windows #840
+* Increase width of tooltips #841
+* Improved fullscreen view in macOS #833
+* Fixed absolute imports in main process #837
```

Comment:
Not user-facing

Reply:
Let's remove it 👌

Reply:
yeah, it's a small change that doesn't affect any functionality. We can remove it